wifi/Makefile

```
@@ -1,12 +1,12 @@
 # name of your application
-APPLICATION = radio-firmware
+APPLICATION = turpial-wifi
```

may be something more generic, `wifi-subsys` or `wifi-subsystem`
```suggestion
APPLICATION = wifi-subsys
```

---

wifi-iface...etc